BPBackendDjango/BPBackendDjango/Views/userviews.py

```diff
@@ -358,6 +391,29 @@ def post(self, request, *args, **kwargs):
             }
             return Response(data)
 
+        #check if arguments are fine
+        if not check_input_length(req_data['first_name'], FIRST_NAME_LENGTH):
+            data = {
+                'success': False,
+                'description': 'first name is too long.',
+                'data': {}
+            }
+            return Response(data)
+        if not check_input_length(req_data['last_name'], LAST_NAME_LENGTH):
+            data = {
+                'success': False,
+                'description': 'first name is too long.',
+                'data': {}
+            }
+            return Response(data)
+        if not check_input_length(req_data['email_address'], EMAIL_LENGTH):
+            data = {
+                'success': False,
+                'description': 'first name is too long.',
```

Comment:
These descriptions are wrong.
As I mentioned in my first review you could create a method like this:
```python
def build_input_length_response(attr_name):
	data = {
		“success“: False,
		“description“: f“{attr_name} too long“,
		“data“: {}	
	}
	return Response(data)
 ```

Then you can use it like this: 
```python
if not check_input_length(req_data['email_address'], EMAIL_LENGTH):
	return build_input_length_response("Email address")
```

That would be way more readable and less duplicate code.